### PR TITLE
testground/infra: create a composition for cluster runner

### LINKED ISSUE #32: infra: connect to Influx DB and populate data
After we finish some details for #25 we need to execute a dry-run with a basic test scenario to see what actually Influx DB data is populated for further analysis

### PR DESCRIPTION
**Pull request by Bidon15**
Adding a composition run for the K8s cluster
Closes #32

### CODE REVIEW

**Comment by Bidon15** [comment]
This should be reviewed after #27

File: init_validators.go [hunk 1]
```diff
@@ -209,6 +209,38 @@ func initVal(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
...
+	var persPeers []string
+	for i := 0; i < runenv.TestGroupInstanceCount; i++ {
+		val := <-valCh
+		runenv.RecordMessage("Validator Received: %s, %s", val.IP, val.PubKey)
+		if !val.IP.Equal(config.IPv4.IP) {
+			persPeers = append(persPeers, fmt.Sprintf("%s@%s", val.PubKey, val.IP.To4().String()))
+		}
+	}
+
+	err = appkit.AddPersistentPeers(configPath, persPeers)
```
Comment (Bidon15) [inline init_validators.go:239]: This should be removed after we get this done https://github.com/celestiaorg/test-infra/issues/40

File: init_validators.go [hunk 2]
```diff
@@ -209,6 +209,38 @@ func initVal(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
...
+	if err != nil {
+		return err
+	}
+
+	valCh := make(chan *appkit.ValidatorNode)
+	client.Subscribe(ctx, testkit.ValidatorPeerTopic, valCh)
+
+	var persPeers []string
+	for i := 0; i < runenv.TestGroupInstanceCount; i++ {
+		val := <-valCh
```
Comment (Wondertan) [inline init_validators.go:232]: 🚓

**Review by Wondertan — APPROVED** [review]
LGTM

File: init_validators.go (hunk 2 above)
Comment (Bidon15) [reply init_validators.go]: Fixed

**Review by sysrex — APPROVED** [review]
lgtm

**Pull request merged**